benchmark/fri/fri_benchmark.cc

```diff
@@ -49,11 +50,16 @@ void Run(const FRIConfig& config) {
   using ExtF = math::BabyBear4;
   using PackedF = math::PackedBabyBear;
   using ExtPackedF = math::PackedBabyBear4;
-  using Poseidon2 = crypto::Poseidon2Sponge<crypto::Poseidon2ExternalMatrix<
-      crypto::Poseidon2HorizenExternalMatrix<F>>>;
-  using PackedPoseidon2 =
+  using Params = crypto::Poseidon2Params<F, 15, 7>;
+  using PackedParams = crypto::Poseidon2Params<PackedF, 15, 7>;
```

Comment:
If the meaning of the second value is `rate`, it appears to be 15, and if it means `width`, it appears to be 16. Which one is correct? In `Poseidon2Params`, it is called `width`.

```suggestion
  using Params = crypto::Poseidon2Params<F, 16, 7>;
  using PackedParams = crypto::Poseidon2Params<PackedF, 16, 7>;
```

Reply:
Actually in `Poseidon2Params` its `rate`! 
``` c++
template <typename _Field, size_t _Rate, uint32_t _Alpha, size_t _Capacity = 1,
          size_t _FullRounds = 8,
          size_t _PartialRounds =
              (GetPoseidon2PartialRounds<_Field, _Rate + _Capacity, _Alpha>())>
```
it should be 15~

Reply:
Oh, that's right, I see. Thank you!